doc/lua_api.txt

```diff
@@ -7350,8 +7350,9 @@ Player properties need to be saved manually.
         -- Collide with other objects if physical = true
 
         collisionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},  -- Default
-        selectionbox = {-0.5, 0.0, -0.5, 0.5, 1.0, 0.5},
+        selectionbox = {rotate = false; -0.5, 0.0, -0.5, 0.5, 1.0, 0.5},
         -- Selection box uses collision box dimensions when not set.
+        -- If `rotate = true` the box will be rotated by the objects rotation (plus the attachment rotation on clients)
```

Comment:
Is it clear enough that setting `rotate = true` in the collisionbox def won't have any effect?